### ISSUE TITLE
Standalone app rejected

### ISSUE THREAD

**Issue by nicolastilly** [Simbul/baker#1200]
Hello everyone, my app has been rejected by Apple. I do not understand what I must do to resubmit my application.
Thank you for your help.

Here is the email from Apple:

> Reasons
> 
> 2.21: Apps that are simply a song or movie should be submitted to the iTunes store. Apps that are simply a book should be submitted to the iBookstore
> 2.21 
> 
> Your app is primarily a book and is therefore not appropriate for the App Store, as noted in the App Store Review Guidelines.
> 
> Books should be submitted to iBooks. To work with Apple on iBooks distribution, first verify that your content meets the following requirements:
> - ISBNs are required for all paid titles you intend to distribute
> - Is in either:
> - EPUB format, passing EpubCheck 1.0.5  
> - Multi-Touch format, created by US ISBN Agency. Then complete the iTunes Connect Online Application.
> 
> Note: An ISBN is not required if you have a free book agreement and choose to offer your book free in iBooks.
> 
> Visit iBooks Aggregators for information on working with the iBooks aggregators.

**Comment by folletto** [comment]
Apple doesn't accept "applications that are just books" (even if, as everything related to Apple approval process, it's not 100% enforced). What Apple tells you is to not build an app but create an ebook instead (EPUB or iBook Author).

The simple answer to that is: it shouldn't be just a book, as Apple mentions. Add interactive, multimedia and animated content, in order to have something that isn't similar to a book that can be simply published on the iBooks Store.

For more details, use the search at the top to read through other people that had the same issue with guideline 2.21. :)

**Comment by nicolastilly** [comment]
The answer is rather strange because I've already made an app with Baker and it was accepted! It was a book with videos: http://itunes.apple.com/app/ecriture-videoludique-n-1/id570244552?mt=8
I proceeded in the same way and the application has not been validated ...

**Comment by folletto** [comment]
Yep. As said, Apple's approval process has a high degree of variability. The same book at a different time with a different reviewer may pass.

**Comment by nin9creative** [comment]
I agree with @folletto - it seems as if there is broad array of "interpretations" of what should be accepted or rejected during the review process.  Does your book have enough "interactivity" that goes beyond what can be accomplished in iBooks?
